tracing-subscriber/src/registry/sharded.rs

```diff
@@ -0,0 +1,230 @@
+use sharded_slab::{Guard, Slab};
+
+use crate::{
+    fmt::span::SpanStack,
+    registry::{LookupSpan, SpanData},
+    sync::RwLock,
+};
+use serde_json::{json, Value};
```

Comment:
:(

Reply:
i know :(

at the moment, I'm focused on filling out the rest of this module, particularly the inter-span relationships.